internal/exec/engine.go

```diff
@@ -103,6 +110,41 @@ func (e Engine) Run(stageName string) error {
 	return nil
 }
 
+// isConfigCacheExist checks if we already have a cached config.
+// TODO(sohan): This is a hack to emit the journald message only
+// once for the base config (https://github.com/coreos/ignition/issues/950)
+func (e *Engine) isConfigCacheExist(cache string) error {
```

Comment:
The naming of this function seems off to me. It seems like what it is doing is `logToJournalIfNoCacheExists()` rather than what is implied by `isConfigCacheExist()` which makes me think it's just a function to return a bool based on if the cache exists or not.

I think we should either change this function name to indicate what it's really doing OR we should change this function to simply be a true/false on if a cache exists or not and we should call `logStructuredJournalEntry()` from inside of `Run()`